### PR TITLE
add names to unique constraints initial schema

### LINKED ISSUE #6417: [Feature] Add uniqueConstraintName on liquibase's initial schema constraints to make easy edit them
##### **Overview of the feature**

Add uniqueConstraintName on liquibase's initial schema constraints to identify them independently of used database.

##### **Motivation for or Use Case**

I tried to drop email unique constraint "jhi_user_email_key" with liquibase and this worked with dev profile (Postgresql).

`		<dropUniqueConstraint constraintName="jhi_user_email_key"
			tableName="jhi_user" />`

When I launched tests, they failed because in H2, these constraint have a diferente name.

If we add a uniqueConstraintName we can drop it without problems and cross-database compatibility.

##### **Reproduce the error**

You can see a example here:
https://stackoverflow.com/questions/46237641/jhipster-integration-tests-fail-when-i-used-liquibase-to-change-model-data/

##### **Related issues**

https://github.com/jhipster/generator-jhipster/issues/6416

##### **Suggest a Fix**

Add these changes on the generator.

##### **JHipster Version(s)**

4.8.2

##### **JHipster configuration**

```
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.company.name",
      "nativeLanguage": "es"
    },
    "jhipsterVersion": "4.8.0",
    "baseName": "name",
    "packageName": "com.company.name",
    "packageFolder": "com/company/name",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "hibernateCache": "no",
    "clusteredHttpSession": false,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "postgresql",
    "prodDatabaseType": "postgresql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "gradle",
    "enableSocialSignIn": false,
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "secret",
    "clientFramework": "angularX",
    "useSass": true,
    "clientPackageManager": "yarn",
    "applicationType": "monolith",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "es",
    "languages": [
      "es"
    ]
  }
}
```

##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

No related.

##### **Browsers and Operating System**

Chrome and Fedora 26

- [X] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by duderoot**
all unique constrains in initial schema should have same name on all db

Fixes #6417

- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by duderoot** [comment]
reopen in order to rerun the build

File: package.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "eslint-plugin-import": "2.7.0",
     "fs-extra": "4.0.2",
     "mocha": "3.5.3",
+    "webdriver-manager": "^12.0.6",
```
Comment (deepu105) [inline package.json]: This is unrelated and I don't understand why its required? That too on the base

Comment (duderoot) [reply package.json]: The build was failing because of the missing selenium lib, which I thing it's had just been fix by @pascalgrimaud  in #6474. On my dev machine I have the webdriver-manager installed globally and the test was passing. In package.json the webdriver-manager was missing and I was hopping to fix the build, which was obviously the wrong approach. I will revert the last commit immediately. The test are gone still not pass until the selenium bug is fixed. Should I wait until the selenium bug is fixed and then rebase on that or what will be the better approach. 10x

Comment (deepu105) [reply package.json]: You can revert this as @pascalgrimaud solution is the right one for this

**Comment by deepu105** [comment]
I think you reverted the wrong commit

**Comment by duderoot** [comment]
:confounded:  :see_no_evil:  just fixed the revert and commited
